tests/unit/submissionworker/test_worker.py

```diff
@@ -0,0 +1,189 @@
+import boto3
+import os
+import shutil
+import tempfile
+import zipfile
+
+from datetime import timedelta
+from moto import mock_sqs
+from os.path import join
+
+from django.core.files.uploadedfile import SimpleUploadedFile
+from django.contrib.auth.models import User
+from django.utils import timezone
+
+from allauth.account.models import EmailAddress
+from rest_framework.test import APITestCase, APIClient
+
+from challenges.models import (
+    Challenge,
+    ChallengePhase,
+)
+from participants.models import Participant, ParticipantTeam
+from hosts.models import ChallengeHostTeam
+from jobs.models import Submission
+
+from scripts.workers.submission_worker import (
+    download_and_extract_file,
+    create_dir,
+    create_dir_as_python_package,
+    return_file_url_per_environment,
+    get_or_create_sqs_queue
+)
+
+
+class BaseAPITestClass(APITestCase):
+
+    def setUp(self):
+
+        self.client = APIClient(enforce_csrf_checks=True)
+
+        self.user = User.objects.create(
+            username='someuser',
+            email="user@test.com",
+            password='secret_password')
+
+        EmailAddress.objects.create(
+            user=self.user,
+            email='user@test.com',
+            primary=True,
+            verified=True)
+
+        self.user1 = User.objects.create(
+            username='someuser1',
+            email="user1@test.com",
+            password='secret_password1')
+
+        EmailAddress.objects.create(
+            user=self.user1,
+            email='user1@test.com',
+            primary=True,
+            verified=True)
+
+        self.client.force_authenticate(user=self.user1)
+
```

Comment:
Please remove unnecessary newlines

Reply:
I have removed a lot of the django code as well, since it was not strictly necessary for the tests. PTAL.